docs/library/network.rst

```diff
@@ -276,18 +276,30 @@ For example::
         argument is passed. Otherwise, query current state if no argument is
         provided. Most other methods require active interface.
 
-    .. method:: wlan.connect(ssid, password)
+    .. method:: wlan.connect(ssid, password, [bssid])
 
         Connect to the specified wireless network, using the specified password.
 
+        The `bssid` argument, if specified, restricts connection to an AP with
+        given hardware address.
```

Comment:
Yes, I think for WPA2, the actual secret is derived from the passphrase AND the ssid.
So if SSID does not match the hidden one, connection will not succeed even if passphrase is correct.